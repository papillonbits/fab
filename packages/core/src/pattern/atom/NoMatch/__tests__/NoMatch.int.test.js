import { regular } from './NoMatch.int.story'

describe('<NoMatch />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
