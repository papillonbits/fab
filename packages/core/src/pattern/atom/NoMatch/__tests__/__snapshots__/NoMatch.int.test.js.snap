// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> Render must match regular 1`] = `
<div>
  <div
    class=""
  >
    <h3>
      No match for route. Return to 
    </h3>
  </div>
</div>
`;
